SignalServiceKit/src/Account/TSAccountManager.m

```diff
@@ -104,9 +105,9 @@ - (void)didRegister
 
     [self storeLocalNumber:phoneNumber];
 
-    [[NSNotificationCenter defaultCenter] postNotificationName:kNSNotificationName_RegistrationStateDidChange
-                                                        object:nil
-                                                      userInfo:nil];
+    [[NSNotificationCenter defaultCenter] postNotificationNameAsync:kNSNotificationName_RegistrationStateDidChange
```

Comment:
Many components use this to update to reflect registration. Can be safely delivered later.